metabase/metabase
# Migrate UserPasswordForm to formik

Epic https://github.com/metabase/metabase/issues/26178

How to test:
- Account -> Password
- Make sure it is possible to update your password
- Make sure you get validation messages for your new password

<img width="580" alt="Screenshot 2022-11-03 at 13 33 40" src="https://user-images.githubusercontent.com/8542534/199710536-35152620-527f-4232-8322-6ddd92ad7a89.png">
